lib/active_set.rb

```diff
@@ -70,6 +72,7 @@ def reinitialize(processed_set, method, instructions)
     self.class.new(@set,
                    view: processed_set,
```

Comment:
We pass both the `@set` and the `processed_set` back in when initializing a new instance of ActiveSet to keep the original set around. This is primarily a convenience, and isn't essential to any operations.